src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -50,6 +49,7 @@ public function process(ContainerBuilder $container)
         $groupDefaults = $admins = $classes = [];
 
         $pool = $container->getDefinition('sonata.admin.pool');
+        $defaultController = $container->getParameter('sonata.admin.configuration.default_controller');
 
```

Comment:
We could add a check here or in `Configuration.php` to check if this class exists and have a better DX for the User?

Reply:
I was thinking that I guess this could be a service name too.